### PR TITLE
Add salary payment in tax module

### PR DESCRIPTION
**Pull request by aspangaro**
This update add a link to an user for salary payment.

Need for future hrm module and in accountancy to detect specific account dedicated to salary.

### CODE REVIEW

**Comment by BebZ** [comment]
Hey Alex, very effective step forward :) 
In your module, or maybe in Dolibarr if it does exist already : 
How would you implement (record ) a subvention (payment) from the governement on a salary ?
(in French it is a C.U.I. C.I.E., sort of "sponsored type of employment contract", for other nations that believe France can't make social dumping !)

Ahaha :)

File: htdocs/compta/salaries/class/salaries.class.php [hunk 1]
```diff
@@ -0,0 +1,578 @@
...
+ */
+
+// Put here all includes required by your class file
+require_once DOL_DOCUMENT_ROOT .'/core/class/commonobject.class.php';
+
+
+/**
+ *  Put here description of your class
+ */
+class Sal extends CommonObject
```
Comment (marcosgdf) [inline htdocs/compta/salaries/class/salaries.class.php]: Why did you call it `Sal`?

**Comment by aspangaro** [comment]
@Hubz Hello,

To manage social benefits, the part "social contribution" in tax module is enough. I think that the identification by user is not necessary.

I added this code to the module tax to be able to identify the code 421 (in France) in accounting.

@marcosgdf Hello,

Sal for Salaries, it's just an abbreviation but I am going to go back on this point.

**Comment by aspangaro** [comment]
@marcosgdf 

It is noted. You were right to specify it to me. I modified as a consequence my piece of code with name of table / class a little more explicit.

**Comment by marcosgdf** [comment]
Great. Also you should check the travis build because there's some code not following the coding style guidelines

**Comment by aspangaro** [comment]
Thanks @marcosgdf  for your time ! 

It's done, I correct the error detects by Travis

**Pull request merged**